app/src/main/java/org/stepik/android/view/in_app_web_view/InAppWebViewDialogFragment.kt

```diff
@@ -116,6 +118,18 @@ class InAppWebViewDialogFragment : DialogFragment(), InAppWebViewView {
         centeredToolbarTitle.text = title
         centeredToolbar.setNavigationOnClickListener { dismiss() }
         centeredToolbar.setTintedNavigationIcon(R.drawable.ic_close_dark)
+        centeredToolbar.inflateMenu(R.menu.in_app_web_view_menu)
+        centeredToolbar.setOnMenuItemClickListener { menuItem ->
+            when (menuItem.itemId) {
+                R.id.menu_item_external -> {
+                    startActivity(Intent(Intent.ACTION_VIEW, Uri.parse(url)))
+                    dismiss()
```

Comment:
can it lead to state loss exception?

Reply:
Similar to [this](https://github.com/StepicOrg/stepik-android/blob/a533c4180ec8d0f16320ac9e9a8c3b7ae5192187/app/src/main/java/org/stepik/android/view/magic_links/ui/dialog/MagicLinkDialogFragment.kt#L67), should be fine.